projects/schematics/src/shared/constants.ts

```diff
@@ -305,6 +307,13 @@ export const ONLY_NUMBER_DIRECTIVE_MODULE = 'OnlyNumberDirectiveModule';
 export const ONLY_NUMBER_DIRECTIVE = 'OnlyNumberDirective';
 export const AUTO_FOCUS_DIRECTIVE = 'AutoFocusDirective';
 export const AUTO_FOCUS_DIRECTIVE_MODULE = 'AutoFocusDirectiveModule';
+
+export const StoreFinderActions = 'StoreFinderActions';
```

Comment:
Perhaps change StoreFinderActions to STORE_FINDER_ACTIONS to be consistent with the rest.

Reply:
🙈  my mistake